debian/ubuntu-advantage-tools.postinst

```diff
@@ -205,17 +227,12 @@ EOF
 
 configure_esm() {
     rm -f $APT_TRUSTED_KEY_DIR/ubuntu-esm*gpg  # Remove legacy esm keys
-    if [ "14.04" = "${VERSION_ID}" ]; then
-            enable_apps="false"
-    else
-            enable_apps="true"
+    if check_is_active_esm "${UBUNTU_CODENAME}"; then
+        install_esm_apt_key_and_source "infra" "$UBUNTU_CODENAME"
     fi
-    if [ "${enable_apps}" = "true" ]; then
-        if ! check_service_is_beta esm-apps; then
-            install_esm_apt_key_and_source "apps" "$UBUNTU_CODENAME"
-        fi
+    if ! check_service_is_beta esm-apps; then
```

Comment:
So currently, if you want to use esm-apps, you'll need to configure allow_beta _before installing_ for esm-apps to get set up - is that right? I just want to call that out to make sure it is what we want here.

Reply:
Correct assumption.
in order to test this on pkg install we'd need to add features:\n  allow_beta: true to `/etc/ubuntu-advantage/uaclient.conf` prior to this package install. This can be done easily in the pkg ugprade from existing system, or at launch time with
```yaml
#cloud-config    
write_files:
 - path: /etc/ubuntu-advantage/uaclient.conf
   content: |
        features:
            allow_beta: true
   append: true